lua/starfall/libs_sv/prop_sent.lua

```diff
@@ -0,0 +1,1595 @@
+-- Simple helper function that provides model if it doesnt exist
+local function registerSent(class, data)
+	data[1].model = data[1].model or {"Model", TYPE_STRING, "models/props_junk/watermelon01.mdl"}
+	
+	list.Set("starfall_creatable_sent", class, data)
+end
+
+local function tryCastColor(tbl, allow_alpha)
```

Comment:
Id just name this castColor. 'try' implies that it could fail and this implementation will never fail.